woodwork/table_accessor.py

```diff
@@ -111,6 +111,8 @@ def init(self,
                 extra_params.append('logical_types')
             if already_sorted:
                 extra_params.append('already_sorted')
+            if not use_standard_tags or isinstance(use_standard_tags, dict):
+                extra_params.append('use_standard_tags')
```

Comment:
now that `use_standard_tags` is no longer a part of the kwargs for the TableAccessor (bc it's default is different than the TableSchema's), we need to add it to this check explicitly